internal/module/kernelmapper.go

```diff
@@ -120,6 +120,7 @@ func (kh *kernelMapperHelper) prepareModuleLoaderData(mapping *kmmv1beta1.Kernel
 	mld.Selector = mod.Spec.Selector
 	mld.ServiceAccountName = mod.Spec.ModuleLoader.ServiceAccountName
 	mld.Modprobe = mod.Spec.ModuleLoader.Container.Modprobe
+	mld.ModuleVersion = mod.Spec.ModuleLoader.Container.Version
```

Comment:
Should we set a default value in case it was not set?

Reply:
This is something that i discussed with @qbarrand in previous PR, but it seems that currently his suggestion is better, since it allows customer who does not want to use ordered upgrade to completely ignore it